.github/actions/mount-s3/action.yml

```diff
@@ -9,6 +9,11 @@ runs:
         sudo apt install s3fs
         sudo mkdir /s3bucket
         sudo chown -R $(id -u):$(id -g) /s3bucket
+        mkdir $HOME/.aws
+        printf "[default]\naws_access_key_id=${AWS_ACCESS_KEY_ID}\naws_secret_access_key=${AWS_SECRET_ACCESS_KEY}\n" >> $HOME/.aws/credentials
+        [[ -v AWS_SESSION_TOKEN ]] && printf "aws_session_token=${AWS_SESSION_TOKEN}" >> $HOME/.aws/credentials
+        cat $HOME/.aws/credentials
+        chmod 600 ${HOME}/.aws/credentials
```

Comment:
No tak načtení credentials z env vars zjevně nefunguje. Tak jsem to změnil na credentials file. Udělal jsem na to dočasný testovací job ve kterým jsem otestoval že to fakt funguje. 😅

![CleanShot 2022-10-16 at 10 27 42@2x](https://user-images.githubusercontent.com/215660/196026178-8e60a1ab-1f2e-4534-9870-43e1e0f8ca2b.jpg)

Auth přes assumnutou roli vypadá takto (je tam session token):

![CleanShot 2022-10-16 at 10 27 12@2x](https://user-images.githubusercontent.com/215660/196026209-01fbbdab-eeb2-406a-b60f-a2de1d0d3a37.jpg)

A auth přes legacy credentials:

![CleanShot 2022-10-16 at 10 29 19@2x](https://user-images.githubusercontent.com/215660/196026242-0c220caf-2652-433a-83a3-edcdd1ac3a29.jpg)

Reply:
ok

Reply:
Ještě jsem oddělal zapomenutej `cat` toho credentiasl filu. 🙂

Reply:
jo koukam ze ty env fakt asi nepodporuje https://github.com/s3fs-fuse/s3fs-fuse#examples, takze ok

Reply:
No právě že podle kódu by měl 🙂  https://github.com/s3fs-fuse/s3fs-fuse/blob/9a4282208ebdcadae44f7caa1f31635abec5dd7f/src/s3fs_cred.cpp#L918-L948 ale to právě nefungovalo.